### PR TITLE
Update to v2.7.0

### PR DESCRIPTION
**Pull request by williamkbentley**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -230,26 +230,43 @@
...
-        "${workspaceFolder}/interactive-app/simple-viewer-app/lib/backend/main.js"
+        "${workspaceFolder}/interactive-app/simple-viewer-app/lib/backend/electronmain.js",
+        "--remote-debugging-port=9223"
       ],
-      "windows": {
-        "runtimeExecutable": "${workspaceFolder}/interactive-app/simple-viewer-app/node_modules/.bin/electron.cmd"
-      }
+      "env": {
+        "NODE_ENV": "development",
+        "ITWIN_VIEWER_SNAPSHOT": "tpb.ibim"
```
Comment (calebmshafer) [inline .vscode/launch.json:247]: Let's leave the iModel empty here or remove this line entirely.

File: .vscode/launch.json [hunk 2]
```diff
@@ -230,26 +230,43 @@
       ]
     },
     {
-      "type": "node",
+      "name": "[BACKEND] simple-viewer-app electron",
```
Comment (calebmshafer) [inline .vscode/launch.json:233]: We should probably make the same changes for the Ninezone app as well.

File: interactive-app/ninezone-sample-app/src/backend/electronmain.ts [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  ElectronRpcManager.initializeImpl({}, rpcs);
+
+  const opts: ElectronManagerOptions = {
+    webResourcesPath: path.join(__dirname, "..", "..", "..", "build"),
+  }
+  const manager = (process.env.NODE_ENV === "development") ? new WebpackDevServerElectronManager(opts) : new IModelJsElectronManager(opts);
+  await manager.initialize({
+    width: 1280,
+    height: 800,
+    webPreferences: {
```
Comment (calebmshafer) [inline interactive-app/ninezone-sample-app/src/backend/electronmain.ts:43]: The recommendation is to now remove both of these webPreferences when starting an Electron app.

File: interactive-app/simple-viewer-app/src/backend/electronmain.ts [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  ElectronRpcManager.initializeImpl({}, rpcs);
+
+  const opts: ElectronManagerOptions = {
+    webResourcesPath: path.join(__dirname, "..", "..", "..", "build"),
+  }
+  const manager = (process.env.NODE_ENV === "development") ? new WebpackDevServerElectronManager(opts) : new IModelJsElectronManager(opts);
+  await manager.initialize({
+    width: 1280,
+    height: 800,
+    webPreferences: {
```
Comment (calebmshafer) [inline interactive-app/simple-viewer-app/src/backend/electronmain.ts:43]: Remove here too.

**Review by calebmshafer — CHANGES_REQUESTED** [review]
Few comments based no the new recommended settings for Electron.

File: interactive-app/simple-viewer-app/src/backend/electronmain.ts (hunk 4 above)
Comment (williamkbentley) [reply interactive-app/simple-viewer-app/src/backend/electronmain.ts]: done

**Review by calebmshafer — APPROVED** [review]
LGTM

**Pull request merged**